### PR TITLE
Get PYTHONPATH from .env file.

### PR DESCRIPTION
**Pull request by astenman**
I had a problem with unresolved references when I was running lsp-python-ms from a subdirectory of the project root. A possible solution to this problem is to put a ".env" file in the project root and define the environment variable "PYTHONPATH" there. This PR adds functionality that parses the ".env" file for PYTHONPATH definitions and sets the environment variable prior to starting the language server.

### CODE REVIEW

File: lsp-python-ms.el [hunk 1]
```diff
@@ -280,6 +281,21 @@ directory"
...
+	 (file (concat (file-name-as-directory root) ".env"))
+	 (rx (concat "^[:blank:]*" envvar "[:blank:]*=[:blank:]*"))
+	 val)
+    (when (file-exists-p file)
+      (with-temp-buffer
+	(insert-file-contents file)
+	(keep-lines rx (point-min) (point-max))
+	(when (string-match (concat rx "\\(.*\\)") (buffer-string))
+	  (setq val (match-string 1 (buffer-string)))
+	  (when (not (string-empty-p val))
```
Comment (seagle0128) [inline lsp-python-ms.el]: It's better use `unless` here.

Comment (seagle0128) [inline lsp-python-ms.el]: Please use SPACE instead of TAB while formating the codes.

**Review by seagle0128 — CHANGES_REQUESTED** [review]

File: lsp-python-ms.el (hunk 1 above)
Comment (astenman) [reply lsp-python-ms.el]: Of course. I forgot to change this setting in my merge tool.

**Comment by seagle0128** [comment]
Looks fine for me. Thanks so much!

**Pull request merged**